README.md

```diff
@@ -1,2 +1,67 @@
 # oop-school-library
-building a school library app with Ruby programming language
+building a oop school library app with Ruby programming language
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+Additional description about the project and its features.
+
+## Built With
+
+- Ruby
+## Getting Started
+To get a local copy up and running, follow the steps in Setup below.
+
+### Prerequisites
+Basic knowledge of Ruby
+
+### Setup
+If you installed git you can clone the code to your machine, or download a ZIP of all the files directly.
+
+[Download the ZIP from this location](https://github.com/topeogunleye/oop-school-library/archive/refs/heads/main.zip), or run the following [git](https://git-scm.com/downloads) command to clone the files to your machine:
+
+```bash
+git clone https://github.com/topeogunleye/oop-school-library.git
+```
+
+- Once the files are on your machine, open the _todo-list_ folder in [Visual Studio Code](https://code.visualstudio.com/), and follow the steps in Installation & Set Up
```

Comment:
- [OPTIONAL] Kindly update your `readme file` to reflect the changes for this milestone. This milestone is about `School library` and not `todo_list`. Please kindly fix that, as this will make your `readme file` more informative and professional.